### PR TITLE
#PS-1317 fix PS-1527 UI Improvement

### PR DESCRIPTION
**Pull request by shreyas1434shinde**

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "AS_OF_LAST_WEEK": "As of last week",
     "LAST_SEVEN_DAYS": "Last 7 Days",
     "LAST_MONTH": "Last Month",
-    "LAST_SIX_MONTHS": "Last Six Months",
+    "Last Six Months": "Last Six Months",
```
Comment (itsvick) [inline public/locales/en/common.json:43]: @shreyas1434shinde why u changed key here?

File: src/assets/images/loginImage.febfbcca.jpg
Comment (itsvick) [inline src/assets/images/loginImage.febfbcca.jpg:1]: @shreyas1434shinde can u update the name of the image

File: src/components/CohortSelectionSection.tsx [hunk 2]
```diff
@@ -328,7 +328,7 @@ const CohortSelectionSection: React.FC<CohortSelectionSectionProps> = ({
                             onChange={handleCohortSelection}
                             displayEmpty
                             inputProps={{ 'aria-label': 'Without label' }}
-                            className="SelectLanguages fs-14 fw-500 bg-white"
+                            className="SelectLanguages capitalize fs-14 fw-500 bg-white"
```
Comment (itsvick) [inline src/components/CohortSelectionSection.tsx:331]: @shreyas1434shinde can u update the SelectLanguages classname with kebab case

File: src/styles/globals.css [hunk 3]
```diff
@@ -995,4 +995,17 @@ legend.Mui-focused {
   overflow-y: auto;
   padding-left: 16px;
   padding-right: 16px;
+}
+
+.loginImg{
```
Comment (itsvick) [inline src/styles/globals.css:1000]: @shreyas1434shinde use kebab case

File: public/locales/en/common.json (hunk 1 above)
Comment (shreyas1434shinde) [reply public/locales/en/common.json:43]: @itsvick resolved

File: src/assets/images/loginImage.febfbcca.jpg
Comment (shreyas1434shinde) [reply src/assets/images/loginImage.febfbcca.jpg:1]: @itsvick resolved

File: src/components/CohortSelectionSection.tsx (hunk 2 above)
Comment (shreyas1434shinde) [reply src/components/CohortSelectionSection.tsx:331]: @itsvick resolved

File: src/styles/globals.css (hunk 3 above)
Comment (shreyas1434shinde) [reply src/styles/globals.css:1000]: @itsvick  resolved

**Pull request merged**